### ISSUE TITLE
Project Proposal: Erie Climate Change Data Visualization

### ISSUE THREAD

**Issue by am0eba-byte** [newtfire/digitProjectDesign-Hub#57]
The effects of climate change encroach unto the Erie area in much slower, indirect ways when compared to other places around the globe – which makes ignoring the threat of climate change easier for our community, perpetuating a tendency for short-term thinking and decision-making in local leadership and government. This project aims to give viewers a glimpse into the future of Erie in this global climate change crisis. 

To bring the problems and complications of global climate change closer to home, we will integrate local environmental data on the past, present, and future of Erie amid climate change, and using that data to inform the creation of vivid digital media imagery and data visualizations about the future of Erie’s natural and infrastructural environment if anthropogenic climate change continues at the rate it is going.  

The ultimate outcome in light of said pursuits will be to create vivid and interactive data visualizations and models with code using the data and projections collected by various local environmental protections agencies. Here are some of the resources I've found for collecting this data so far:

[Sea Grant PA Climate Impacts to Erie](http://seagrant.psu.edu/topics/erie-climate-change/projects/climate-impacts-erie)

[Projected Stream Flows under Climate Change Scenarios study](http://seagrant.psu.edu/topics/research/probabilistic-characterization-stream-flows-urbanizing-basins-pennsylvania-under)

[Great Lakes Climate Change Impacts Summary](https://elpc.org/wp-content/uploads/2020/04/2019-ELPCPublication-Great-Lakes-Climate-Change-Report.pdf)

[Erie is ranked 8th in America's fastest-warming cities](https://www.climatecentral.org/news/report-american-warming-us-heats-up-earth-day)

[Climate Change at Presque Isle article](https://www.goerie.com/entertainmentlife/20190929/planning-presque-isle-state-parks-future-amid-climate-change)

[GoErie Article](https://www.goerie.com/news/20190505/eries-climate-is-changing)

**Comment by BarbieCessar** [comment]
I really like this idea! Climate Change is a real problem that I don't think gets enough attention. I like how this project would focus specifically on the Erie area. This is definitely a project I could see myself joining.

**Comment by san5281** [comment]
I am very interested in the amount of deforestation that inevitably came as Erie became increasingly urban as it has a direct relation with global warming.

**Comment by Tiny-Pickles** [comment]
I really like this idea! I think creating a interactive map that encompasses all of lake Erie would really open up our eyes to the importance of our great lake. One of the main issues lake Erie experiences is harmful algal blooms from agricultural/fertilizer runoff, this can have a damaging affect to our ecosystem and biota in our lake. I think we could tie in how some forms of pollution layered with the affects of climate change can have a severe impacts on our lake. I would be really interested in joining this project and there are so many important issues to talk about. 

Here are some additional links that talk about algal blooms and how climate change affects them:
https://www.epa.gov/nutrientpollution/climate-change-and-harmful-algal-blooms
https://oceanservice.noaa.gov/hazards/hab/great-lakes.html

**Comment by AtomicOlsen** [comment]
Super awesome idea here, and I think this project has the potential to really open some eyes. Starting local is the way to go!

**Comment by GHess09** [comment]
I really like this idea!  As a nature photographer, I spend a lot of my time outdoors in Erie, especially at Presque Isle.  At the rate the water level has been rising, the state park will probably be under water someday.  I have a passion for nature and our local area, and could definitely see myself as part of this project!  I can provide lots of different multimedia assets to the project!

**Comment by am0eba-byte** [comment]
@GHess09 That'd be awesome!

**Comment by sxa585** [comment]
Yp meea 

I know we talked about this the other day, but in the GIS classes I took we learned a little bit about local changes in the water and erosion levels around lake Erie area I am not sure about how specific it is or the exact nature of it but I know something is there. Dr. Naber (mdn10@psu.edu) would most likely be one of your main contacts for this. ALso GIS is super cool so that might just be helpful in general.
